docs/iris/src/whatsnew/contributions_2.0/incompatiblechange_2017-Oct-27_logging_removed.txt

```diff
@@ -0,0 +1 @@
+* In Iris v1 it was possible to configure Iris to log at import time through :attr:`iris.config.LOGGING`. This capability has been removed in iris 2.
```

Comment:
"to log at import time": to log _what_ at import time?

"iris 2" --> "Iris v2"